packages/zustand-x/src/types/store.ts

```diff
@@ -0,0 +1,106 @@
+import { StoreMutatorIdentifier } from 'zustand';
+
+import {
+  TCreatedStoreMutateType,
+  TCreatedStoreType,
+  TGetStoreEqualityRecord,
+  TGetStoreRecord,
+  TSetStoreRecord,
+  TState,
+} from './utils';
+
+export type TSelectorBuilder<
+  StateType extends TState,
+  Mutators extends [StoreMutatorIdentifier, unknown][],
+  TActions = {},
+  TSelectors = {},
+> = (
+  set: TStoreApiSet<StateType, Mutators, TActions>,
+  get: TStoreApiGet<StateType, Mutators, TSelectors>,
+  api: TStateApiForBuilder<StateType, Mutators, TActions, TSelectors>
```

Comment:
@imarabinda README has `extendSelectors((state, get, api)`, should we update the doc?